plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/messages.properties

```diff
@@ -1,6 +1,5 @@
+apiclientlibrariespage-description=Additional jars for applications using Google Client APIs for Java
 appengine-title=App Engine Standard Environment Libraries
-appengine-description=Additional jars commonly used in App Engine standard environment applications
 clientapis-title=Google Client APIs for Java
```

Comment:
I prefer "Google Cloud Client Libraries for Java", to be consistent.

![selection_033](https://user-images.githubusercontent.com/10523105/32353165-aa3c1f88-bffa-11e7-8ccd-d2bc4bb4eaaa.png)

Reply:
"Google Cloud Client Libraries for Java" is a very specific group that does not include the Objectify or Endpoints libraries.  Not sure whether that matters here or not.

Reply:
Thoughts on this?  Here are a few options:
  - Google Client APIs for Java
  - GCP Libraries for Java Projects
  - Java APIs for GCP Projects

Reply:
As I said, APIs is overloaded and we shouldn't use it if we can help it.
Maybe just "Google Libraries for GCP" or "Google Cloud Platform Libraries."
I'm sort of mixing together libraries for running on GCP (Objectify, endpoints) with libraries that access GCP.

Reply:
I like "Google Cloud Platform Libraries".